src/components/storage-efficiency-card/storage-efficiency-card.tsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright contributors to the ibm-storage-odf-console project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ import * as React from 'react';
+ import { useTranslation } from 'react-i18next';
+ import { useDashboardPrometheusQuery as usePrometheusQuery } from "@console/dynamic-plugin-sdk/provisional";
+ import {
+  Tooltip,
+} from "@patternfly/react-core";
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+} from '@console/dynamic-plugin-sdk/provisional';
+import OutlinedQuestionCircleIcon from "@patternfly/react-icons/dist/js/icons/outlined-question-circle-icon";
+import { humanizeBinaryBytes } from "../../humanize";
+import { EFFICIENCY_SAVING_QUERY } from "../../constants/queries";
+import './storage-efficiency-card.scss';
+
+const StorageEfficiencyCardBody: React.FC = () => {
+  const { t } = useTranslation();
+  const [saving, ,] = usePrometheusQuery(
+    EFFICIENCY_SAVING_QUERY,
+    humanizeBinaryBytes,
+  );
+  let status = t('Not available');
+  if (saving.value > 0) { 
+    status = saving.string
+  }
+  return (
+    <div className="co-inventory-card__item">
+      <div className="co-utilization-card__item-section-multiline">
+        <h4 className="pf-c-content pf-m-md">{t('Savings')}</h4>
+        <div className="text-secondary">
+          {status}
+          <span className="ibm-storage-efficiency-card-help">
+            <Tooltip            
+              position="top"
+              content={t('The amount of physical storage saved after applying compression.')}
```

Comment:
The internationalization support for dynamic plugins is still in progress.
Expect some changes since we need to have a single namespace file and that namespace should be the prefix in externalization as well. You need to maintain one `src/locales/` folder. 

Example:
```js
     content={t('plugin_ibm-operator~The amount of physical storage saved after applying compression.')}
```
For now you can simply ignore translations, since I dont see you are generating any translation keys for this.

Reply:
localization will be considered in later phase. 
Question about the pre-fix, what is the usage of it? can we name it as whatever string?

Reply:
I agree with @afreen23. For now completely ignore it. We will discuss this once we have both the plugins ready. We have same issue for ODF plugin as well.

Reply:
<img width="483" alt="image" src="https://user-images.githubusercontent.com/76933520/125760650-abe53409-bf09-4471-99cf-5a002f2cfc47.png">

The web page will display the additional string, how to fix it? @afreen23

Reply:
translation will be in another pull request due to https://github.com/openshift/console/pull/9196

Reply:
```suggestion
              content="The amount of physical storage saved after applying compression"
```